tools/abi_downloader/index.ts

```diff
@@ -16,11 +16,30 @@ const allowedABIs = [
   'EIP20CoGateway',
 ];
 
-function writeToFile(name, abi, contractVersion): void {
+/**
+ * This method removes patch contractVersion from the contractVersion if any.
+ * @param contractVersion Version of contract.
+ */
+function getMajorMinorVersion(contractVersion: string): string {
+  // Check if patch version exists.
+  if (contractVersion.split('.').length > 2) {
```

Comment:
It should fail when the length is 1. This check can go in `tools/abi_downloader/run.sh` file.

Reply:
`npm install @openst/mosaic-contracts@0` is valid version of package.

Reply:
True, but we should not allow it. We should have a major-minor version else we cannot differentiate between `v0.10` and `v0.11`

Reply:
In that case, I will add validation for all three version `MAJOR.MINOR.PATCH` and remove this logic here.  OK?